core/sdk/src/api/mod.rs

```diff
@@ -22,6 +22,7 @@ pub mod caps;
 pub mod config;
 pub mod device;
 pub mod manifest;
+pub mod mock_websocket_server;
```

Comment:
There is an existing feature flag for `test`, feel we should add this as part of the feature flag?
Can this be added into `core/tdk` and be available for all test scripts?